pkg/controller/core/workload_controller.go

```diff
@@ -534,8 +537,6 @@ func (r *WorkloadReconciler) triggerDeactivationOrBackoffRequeue(ctx context.Con
 		if err := r.client.Update(ctx, wl); err != nil {
 			return false, err
 		}
-		r.recorder.Eventf(wl, corev1.EventTypeNormal, kueue.WorkloadEvictedByDeactivation,
```

Comment:
Maybe we don't delete this event. It's been already released. It still has some information for marking `active=false`.

Reply:
Yes, but we have it on [here](https://github.com/kubernetes-sigs/kueue/blob/1e874b28b3cefc517c9f7dd799c56863c9c79687/pkg/controller/core/workload_controller.go#L210).

Reply:
So you propose to move [this](https://github.com/kubernetes-sigs/kueue/blob/1e874b28b3cefc517c9f7dd799c56863c9c79687/pkg/controller/core/workload_controller.go#L210-L211) logic to L540?

Reply:
I'm not sure why we separated [event](https://github.com/kubernetes-sigs/kueue/blob/main/pkg/controller/core/workload_controller.go#L537) and [metric](https://github.com/kubernetes-sigs/kueue/blob/main/pkg/controller/core/workload_controller.go#L211) updates.

Reply:
> So you propose to move [this](https://github.com/kubernetes-sigs/kueue/blob/1e874b28b3cefc517c9f7dd799c56863c9c79687/pkg/controller/core/workload_controller.go#L210-L211) logic to L540?

I was proposing to keep it as is (just for backwards-compatibility with 0.7). I'm also ok to drop it (as you did). WDYT @alculquicondor ?

Reply:
I like this implementation.

Reply:
ack